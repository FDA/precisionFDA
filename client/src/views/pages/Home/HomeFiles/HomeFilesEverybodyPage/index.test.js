import React from 'react'
import { shallow } from 'enzyme'

import { HomeFilesEverybodyPage } from './index'


describe('HomeFilesEverybodyPage test', () => {
  it('should render', () => {
    const component = shallow(<HomeFilesEverybodyPage />)

    expect(component).toMatchSnapshot()
  })
})
