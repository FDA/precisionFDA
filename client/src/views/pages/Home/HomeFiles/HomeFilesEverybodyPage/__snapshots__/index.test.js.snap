// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeFilesEverybodyPage test should render 1`] = `
<HomeLayout>
  <div
    className="home-page-layout__header-row"
  >
    <div
      className="home-page-layout__actions"
    />
    <div
      className="home-page-layout__actions--right"
    >
      <Connect(ActionsDropdown)
        deleteFiles={[Function]}
        files={Array []}
        page="public"
      />
    </div>
  </div>
  <div
    className="pfda-padded-t20"
  >
    <Connect(HomeFilesEverybodyTable)
      files={Array []}
      handleFilterValue={[Function]}
    />
  </div>
</HomeLayout>
`;
